torchao/utils.py

```diff
@@ -55,3 +58,10 @@ def benchmark_torch_function_in_microseconds(f, *args, **kwargs):
     )
     measurement = t0.blocked_autorange()
     return measurement.mean * 1e6
+
+
+def find_multiple(n: int, *args: Tuple[int]) -> int:
```

Comment:
why?

Reply:
we now use this in torchao/dtypes and torchao/quantization and have to do import tricks to avoid circular dep